art_web_app/views.py

```diff
@@ -1,5 +1,29 @@
 from django.shortcuts import render
+from artworks.models import Artwork, ArtCategory, Category
+from django.core.exceptions import ObjectDoesNotExist
+import random
 
 
 def home_view(request):
-    return render(request, "home.html", {})
\ No newline at end of file
+    invalid_ids = set()
+    
+    try:
+        nr_of_artworks = Artwork.objects.latest('id').id
+    except ObjectDoesNotExist as e:
+        art = None
+
+    while True:
```

Comment:
Is this for getting a random object within DB? If so, I don't like it. Since you already get all the `Artwork` objects in the code above, you can use `random.choice(Artwork.objects.all())`.